LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIRecruitSoldiers_LWOTC.uc

```diff
@@ -0,0 +1,46 @@
+//---------------------------------------------------------------------------------------
+//  FILE:	 UIRecruitSoldiers_LWOTC.uc
+//  AUTHOR:	 KDM
+//  PURPOSE: Long War of the Chosen compatible recruit screen.
+//--------------------------------------------------------------------------------------- 
+
+class UIRecruitSoldiers_LWOTC extends UIRecruitSoldiers;
```

Comment:
A `_LW` suffix is sufficient and preferred. I'm not that interested in identifiying classes that were added in LWOTC rather than LW2. The suffix is there just to identify vanilla classes that have been Long War-ized. Same goes for the other classes that currently have a `_LWOTC` suffix.